migration/migrating_3_4/deploying-cam.adoc

```diff
@@ -1,12 +1,14 @@
-[id='deploying-cam_{context}']
+[id='deploying-cam_migrating-3-4']
```

Comment:
Same here about keeping these _{context} in the anchor definition. There are a few other places as well.